openhands/runtime/action_execution_server.py

```diff
@@ -811,5 +817,42 @@ async def list_files(request: Request):
             logger.error(f'Error listing files: {e}')
             return []
 
+    @app.get('/view')
```

Comment:
Would this potentially allow an external attacker to read any file on the user's machine, especially when using LocalBox?

Reply:
@openhands-agent can you help me fix this? maybe make this /view only accessible with localhost?

Reply:
@openhands